src/components/ShoppingCart/dropDownButton.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { GrDown, GrUp } from "react-icons/gr";
+import { IconButton } from "@chakra-ui/react";
+import { Collapse } from "@chakra-ui/react";
+
+type quotationDetail = {
+  content: string;
+};
+const DropDownButton = ({ content }: quotationDetail) => {
+  const [show, setShow] = useState(false);
+
+  const handleToggle = () => setShow(!show);
+
+  const scroll = "scroll";
+  const hidden = "hidden";
+  const noWrap = "nowrap";
+  const normal = "normal";
+  const clip = "clip";
+  const ellipsis = "ellipsis";
+
+  return (
+    <div
```

Comment:
Since we use the Charka UI, should we limit the use of HTML tags like <div>?

Reply:
Div tags were replaced by Box from Chakra UI.